presqt/targets/curate_nd/functions/fetch.py

```diff
@@ -5,14 +5,17 @@
 from presqt.targets.curate_nd.utilities import get_curate_nd_resource
 
 
-def curate_nd_fetch_resources(token):
+def curate_nd_fetch_resources(token, search_parameter):
     """
     Fetch all CurateND resources for the user connected to the given token.
 
     Parameters
     ----------
     token : str
         User's CurateND token
+    search_parameter : dict
+        The search parameter passed to the API View
+        Gets passed formatted as {'title': 'search_info'}
```

Comment:
Remember to update the readthedocs section about target integration.

Reply:
👍